### PR TITLE
chore: improve ssl e2e test coverage

### PR DESCRIPTION
**Pull request by starsz**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [x] Improve performance
- [ ] Backport patches

Because some function in SSL e2e test is not covered.
This PR is to cover those functions.
image

### CODE REVIEW

**Comment by starsz** [comment]
cc @Jaycean Can you review my code when you are free?

**Comment by Jaycean** [comment]
> cc @Jaycean Can you review my code when you are free?

https://github.com/apache/apisix-dashboard/pull/1571/checks?check_run_id=2074615023

ci failed. I have also encountered this problem. Need to modify the matching characters.

**Comment by starsz** [comment]
> > cc @Jaycean Can you review my code when you are free?
> 
> https://github.com/apache/apisix-dashboard/pull/1571/checks?check_run_id=2074615023
> 
> ci failed. I have also encountered this problem. Need to modify the matching characters.

Let me found the root cause.

File: api/test/e2enew/ssl/ssl_test.go [hunk 1]
```diff
@@ -26,38 +26,80 @@ import (
...
-		body, err := json.Marshal(map[string]interface{}{
-			"id":   "1",
-			"cert": string(testCert),
-			"key":  string(testKey),
-			"labels": map[string]string{
-				"build":   "16",
-				"env":     "production",
-				"version": "v3",
+	t := ginkgo.GinkgoT()
+	var (
```
Comment (tokers) [inline api/test/e2enew/ssl/ssl_test.go:37]: A better pattern is to put all these initialization logic to `ginkgo.BeforeEach()`, anyway, it's just a suggestion, you can optimize them in the future.

Comment (starsz) [reply api/test/e2enew/ssl/ssl_test.go:37]: Yes. I had used `ginkgo.BeforeEach()` before, but we use those variables as the parameter later.So the `BeforeEach` can't modify those variables.

**Review by nic-chen — APPROVED** [review]

File: api/test/e2enew/ssl/ssl_suite_test.go [hunk 2]
```diff
@@ -25,12 +25,11 @@ import (
...
 )
 
-func TestRoute(t *testing.T) {
+func TestSSL(t *testing.T) {
 	ginkgo.RunSpecs(t, "ssl suite")
 }
 
 var _ = ginkgo.AfterSuite(func() {
 	base.CleanResource("ssl")
-	base.CleanResource("routes")
```
Comment (Jaycean) [inline api/test/e2enew/ssl/ssl_suite_test.go:34]: @starsz @nic-chen 
I have a question here. If we create a route in the test case and delete the route, and verify that the route is indeed deleted, do we still need to executeAfterSuite(base.CleanResource("resource") to ensure the deletion of resources?

Comment (starsz) [reply api/test/e2enew/ssl/ssl_suite_test.go:34]: I think we don't need to do `execute AfterSuite(base.CleanResource("resource") `

Comment (starsz) [reply api/test/e2enew/ssl/ssl_suite_test.go:34]: By the way, I remove the response check. cc @nic-chen 

https://github.com/apache/apisix-dashboard/pull/1571/files#diff-a15775a07088498daed7a85df9b271a05806905c320e38029391c37bef35fb37L277

Comment (nic-chen) [reply api/test/e2enew/ssl/ssl_suite_test.go:34]: ok

Comment (Jaycean) [reply api/test/e2enew/ssl/ssl_suite_test.go:34]: OK

**Review by Jaycean — APPROVED** [review]

**Pull request merged**